pkg/mq/msgstream/mq_msgstream.go

```diff
@@ -649,7 +681,7 @@ func (ms *MqTtMsgStream) bufMsgPackToChannel() {
 
 			msgPack := MsgPack{
```

Comment:
may send a empty msgPack with 0 EndTs and empty Msgs here.
If function `continueBuffering` return false because of 
- ctx.Error
- dynamically modification of config, such as `EnablePursuitMode`. 

A validation of empty msgPack should be applied before sent.

Reply:
- changing `EnablePursuitMode` will not produce empty pack since either way first run will run
- for `ctx.Error` case, msgstream is closing and all consumer shall not consume this pack. I agree to safer to add check here.